agent_shared/src/config.rs

```diff
@@ -185,6 +185,7 @@ pub enum CredentialEvent {
     UnsignedCredentialCreated,
     SignedCredentialCreated,
     CredentialSigned,
+    NotificationReceived,
```

Comment:
Great that you added this!

You can also add it to the `agent_event_publisher_http/README.md` file where we list all our available events (we should at some point try to figure out how we can automate this).